lib/VM/SegmentedArray.cpp

```diff
@@ -121,7 +121,7 @@ SegmentedArray::create(Runtime *runtime, size_type capacity, size_type size) {
     return ExecutionStatus::EXCEPTION;
   }
   PseudoHandle<SegmentedArray> self = std::move(*arrRes);
-  // TODO T25663446: This is potentially optimizable to iterate over the inline
```

Comment:
optimizable seems to be the more common spelling

Reply:
Reverted.